src/index.ts

```diff
@@ -158,6 +169,10 @@ class ClientApp {
      * @param cfg - Runtime config of the client
      */
     constructor(cfg: {
+        /** The origin of the genesys cloud parent. Overrides the pcEnvironment and functions similar to pcOrigin except the gcHostOrigin can be passed by the genesys cloud parent*/
+        gcHostOriginQueryParam?: string;
+        /** The type of environment the genesys cloud parent is running under(e.g. dev, test, prod)*/
+        gcTargetEnvQueryParam?: string;
```

Comment:
maybe for convenience we support these as either queryParams or strings with the string values taking precedence if both happen to be specified.

The queryParam is how we expect most apps to configure and get their config so we provide these as convenience methods.  However, there are cases where an app might need to parse the values from elsewhere or just manually pass in